Tests/Pcap++Test/Tests/FilterTests.cpp

```diff
@@ -656,6 +656,35 @@ PTF_TEST_CASE(TestPcapFiltersOffline)
 	}
 	rawPacketVec.clear();
 
+	//------------------------------
+	// Composite logic filter - Proto + IP
+	//------------------------------
+	{
+		pcpp::IPFilter ipFilter("10.0.0.6", pcpp::SRC);
+		protoFilter.setProto(pcpp::UDP);
+		std::vector<pcpp::GeneralFilter*> filterVec;
+		filterVec.push_back(&ipFilter);
+		filterVec.push_back(&protoFilter);
+		pcpp::CompositeLogicFilter<pcpp::CompositeLogicFilterOp::AND> compositeFilter(filterVec);
```

Comment:
Can't we use directly pcpp:AndFilter here ?

Reply:
Yes, and probably will be, just haven't gotten to it yet. This comment also references the same suggestion: https://github.com/seladb/PcapPlusPlus/pull/1306#discussion_r1485352091

Reply:
Resolved e86630e